### ISSUE TITLE
Create a style guide document with recommended fonts and colors

### ISSUE THREAD

**Issue by Calinou** [godotengine/godot-design#62]
This guide can be distributed as a Markdown file in this repository.

**Comment by fire** [comment]
Are we allowed to include the fonts in this repo?

**Comment by Calinou** [comment]
> Are we allowed to include the fonts in this repo?

Yes, but we just need to link to Google Fonts which has direct downloads available.

**Comment by fire** [comment]
Is this issue still valid?

**Comment by Calinou** [comment]
Yes – we need to create a formal document for this, which will be useful when redesigning the website.

**Comment by Calinou** [comment]
> Any news ?

To my knowledge, nobody is currently working on this.

**Comment by 3Samourai** [comment]
Internal documentation for Godot: just a PDF, but I want a consistent design.

**Comment by fire** [comment]
If you want to help organize our existing documentation I can help.

**Comment by 3Samourai** [comment]
It's where I found this repo 😄
